client-libs/typescript/openai/index.ts

```diff
@@ -0,0 +1,111 @@
+import * as openai from "openai-beta";
+import { readEnv, type RequestOptions } from "openai-beta/core";
+import { CompletionCreateParams } from "openai-beta/resources/chat/completions";
+import axios from "axios";
+
+export * as openai from "openai-beta";
+import * as openPipeClient from "../codegen";
+
+interface ClientOptions extends openai.ClientOptions {
+  openPipeApiKey?: string;
```

Comment:
I think as a caller I'd rather just have a single `openpipe` top-level arg everywhere that our stuff nests inside of.

Reply:
Yes, I agree